cvat-sdk/cvat_sdk/core/downloading.py

```diff
@@ -27,21 +29,52 @@ class Downloader:
     def __init__(self, client: Client):
         self._client = client
 
+    @staticmethod
+    def _get_server_filename(response: urllib3.HTTPResponse) -> str:
+        content_disposition = next(
+            (
+                filename
+                for part in response.headers.get("Content-Disposition", "").split(";")
+                if (filename := part.strip()) and filename.startswith("filename=")
+            ),
+            None,
+        )
+
+        filename = None
+        if content_disposition:
+            filename_parts = content_disposition.split("=", maxsplit=1)
+            if len(filename_parts) == 2:
```

Comment:
It seems like this should always be true, since you already verified that the string contains an equals sign in the comprehension.